examples/third_person_control/run.py

```diff
@@ -0,0 +1,182 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+# Before running this file, rename user_config.yaml.example -> user_config.yaml and modify it with appropriate paths for your system.
+
+import argparse
+import os
+
+import numpy as np
+import pandas as pd
+
+import spear
+
+
+def get_data_frame(skeletal_mesh_transforms):
+    columns = np.array([
+        name + "." + field + "." + axis
+        for name, transform in skeletal_mesh_transforms.items()
+        for field, vector in transform.items()
+        for axis, val in vector.items()
+    ]).ravel()
+    data = np.array([
+        value
+        for name, transform in skeletal_mesh_transforms.items()
+        for field, vector in transform.items()
+        for axis, value in vector.items()
+    ]).reshape(1, -1)
+    return pd.DataFrame(columns=columns, data=data)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--actions_file", default=os.path.realpath(os.path.join(os.path.dirname(__file__), "actions.csv")))
+    args = parser.parse_args()
+
+    np.set_printoptions(linewidth=200)
+
+    # load config
+    config = spear.get_config(user_config_files=[os.path.realpath(os.path.join(os.path.dirname(__file__), "user_config.yaml"))])
+    spear.configure_system(config)
+    instance = spear.Instance(config)
+
+    with instance.begin_frame():
+        gameplay_statics_uclass = instance.unreal_service.get_static_class(class_name="UGameplayStatics")
+        controller_uclass = instance.unreal_service.load_class(class_name="UObject", outer=0,
```

Comment:
`controller_uclass` not used

Reply:
Im not quite clear on why we need to register uclass in advance. My understanding is in somecases we need to know the c++ class for template functions, but I wonder if unreal provide corresponding function with explicit uclass parameter with no need of template in most cases.

Reply:
I think it is important to demonstrate that user can get any arbitary uclass and use them regardless its origin, either from engine or user defined, c++ or blueprint.